lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -1980,13 +2203,14 @@ struct ConvertLayoutOpConversion
   }
 
 private:
-  template <typename T>
-  SmallVector<T> reorder(ArrayRef<T> input, ArrayRef<unsigned> order) const {
+  template <typename T, typename RES_T = T>
+  SmallVector<RES_T> reorder(ArrayRef<T> input, ArrayRef<unsigned> order) const {
```

Comment:
Can we use the general reorder in Utility.h?

Reply:
yeah, done.